package.json

```diff
@@ -4,7 +4,7 @@
   "license": "MIT",
   "volta": {
     "node": "20.15.1",
-    "yarn": "1.22.10"
+    "yarn": "4.3.1"
```

Comment:
Do we want to do 4.4.0?

Reply:
Unless there's a reason not to, I'd say yes, let's use the latest at time of merge.